pkg/client/meta/meta.go

```diff
@@ -18,9 +18,9 @@ import (
 	infinispanv1 "github.com/infinispan/infinispan-operator/pkg/apis/infinispan/v1"
 	grafana "github.com/integr8ly/grafana-operator/v3/pkg/apis/integreatly/v1alpha1"
 	keycloakv1alpha1 "github.com/keycloak/keycloak-operator/pkg/apis/keycloak/v1alpha1"
-	kafkabetav1 "github.com/kiegroup/kogito-cloud-operator/api/kafka/v1beta1"
 	"github.com/kiegroup/kogito-cloud-operator/api/v1beta1"
-	"github.com/kiegroup/kogito-cloud-operator/pkg/logger"
+	kafkabetav1 "github.com/kiegroup/kogito-cloud-operator/core/api/kafka/v1beta1"
+	"github.com/kiegroup/kogito-cloud-operator/core/logger"
```

Comment:
This package (meta) can be moved to `core` or `infrastructure`.

Reply:
done